[DOCS-13661] Document triple curly brace syntax for raw/unencoded output (#35302)
DataDog/documentation: content/en/monitors/notify/variables.md

### Character escape

> Variable content is HTML-encoded by default. For example, special characters like `&` and `'` are converted to `&amp;` and `&#x27;` in the rendered output. To output raw, unencoded content, use triple curly braces instead of double curly braces:

Author: @maycmlee it's not the notification message, it's the template variables, but I had to reread the ticket a few times to understand this myself, so maybe a table showing a user the difference would be better:

```suggestion
Variable content is HTML-encoded by default. To output raw, unencoded content, use triple curly braces instead of double curly braces.

For example, when a variable's value contains a URL with query parameters, the `&` is treated differently between the double and triple brace:

| Syntax | Example output |
--------|----------------|
| `{{template_variable}}` (double braces) | `https://status.example.com/check?service=web&amp;region=us-east` |
| `{{{template_variable}}}` (triple braces) | `https://status.example.com/check?service=web&region=us-east` |
```

> Variable content is HTML-encoded by default. For example, special characters like `&` and `'` are converted to `&amp;` and `&#x27;` in the rendered output. To output raw, unencoded content, use triple curly braces instead of double curly braces:

Reviewer: The table with the examples are great! Just a small suggestion for this part:

> the `&` is treated differently between the double and triple brace:

Suggestion:

> the `&` is treated differently depending on whether double or triple braces are used:

### Character escape

Variable content is HTML-encoded by default. To output raw, unencoded content, use triple curly braces instead of double curly braces.

> For example, when a variable's value contains a URL with query parameters, the `&` is treated differently between the double and triple brace:

Author: ```suggestion
For example, when a variable's value contains a URL with query parameters, the `&` is treated differently depending on whether double or triple braces are used:
```
